pallets/auctions/src/lib.rs

```diff
@@ -194,13 +220,26 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
 		fn on_finalize(now: T::BlockNumber) {
-			Self::conclude_auction(now);
+			Self::conclude_auction(now).unwrap_or_default();
```

Comment:
what are you trying to do? if you want to pass the result of a function to the outer scope use the question mark. unwrap_or_default should be used only when you actually want to pass the default if unwrap fails

Reply:
Note: on finalize cannot fail. And you cannot return any error from there.

Reply:
And you cannot even return weight consumed by on finalized. That's why you must return return the wieght of on finalize ffrom on_initialized. 

Which i cannot see in this pallet ?! 

You need to benchmarks somehow the weight used in on finalized and return it from on_initialized.